tracer/tracer.go

```diff
@@ -1297,16 +1297,26 @@ func (t *Tracer) StartOffCPUProfiling() error {
 		return errors.New("off-cpu program finish_task_switch is not available")
 	}
 
-	kprobeSymbol, err := t.kernelSymbols.LookupSymbolByPrefix("finish_task_switch")
+	kprobeSymbs, err := t.kernelSymbols.LookupSymbolsByPrefix("finish_task_switch")
```

Comment:
Make "finish_task_switch" a const to avoid duplication in L1318.

Reply:
In L1318 `finish_task_switch` is not used as string - like `%s` or part of an argument to a function - but is part of a longer text that needed to be divided to comply with golangci-linter.

Reply:
Added a const with https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/370/commits/45a3b06f6437623dd31ae806da1ef9646951242e.